Lib/test/test_ast.py

```diff
@@ -1326,6 +1326,41 @@ def test_dump_incomplete(self):
         self.assertEqual(ast.dump(node, annotate_fields=False),
             "Raise(cause=Name('e', Load()))"
         )
+        # Arguments:
+        node = ast.arguments(args=[ast.arg("x")])
+        self.assertEqual(ast.dump(node, annotate_fields=False),
+            "arguments([], [arg('x')])",
+        )
+        node = ast.arguments(posonlyargs=[ast.arg("x")])
+        self.assertEqual(ast.dump(node, annotate_fields=False),
+            "arguments([arg('x')])",
+        )
+        node = ast.arguments(posonlyargs=[ast.arg("x")], kwonlyargs=[ast.arg('y')])
+        self.assertEqual(ast.dump(node, annotate_fields=False),
+            "arguments([arg('x')], kwonlyargs=[arg('y')])",
```

Comment:
This is a bit weird (why is `kwonlyargs` still passed as a keyword arg) but it's consistent with how `annotate_fields` already works:

```
>>> ast.dump(ast.arguments(kwonlyargs=[ast.arg("x")]), annotate_fields=False)
"arguments(kwonlyargs=[arg('x')])"
```

So this seems fine.

Reply:
Because of these lines: https://github.com/python/cpython/blob/acf69e09c66f8473399fabab36b81f56496528a6/Lib/ast.py#L148-L150

Basically, if any value between is skipped, we use keywords.